tools/rna_tools/selectsequences/selectsequences.xml

```diff
@@ -0,0 +1,64 @@
+<tool id="SelectSequences" name="SelectSequences" version="1.0.0">
+  <description>
+    Tool to select representative sequences from a multiple sequence alignment.
+    Useful before running RNAz, RNAcode, RNAalifold on alignments with many entries.
+  </description>
+  <requirements>
+    <requirement type="package" version="1.0.0">selectsequences</requirement>
+  </requirements>
+  <command detect_errors="aggressive"><![CDATA[
+    SelectSequences 
+      -c '$input_clustal'
+      #if $x
```

Comment:
You can replace the entire `if` with just `$x`. You are using falsevalue and truevalue in your param, so in case the option is not active it is an empty string, in case of active it will be `-x`